packages/react/src/River/River.module.css

```
@@ -22,6 +22,11 @@
   object-fit: cover;
 }
 
+.River__visual--rounded,
+.River__visual--rounded img {
+  border-radius: var(--brand-borderRadius-large);
+}
```

we might need to add video as well, the posters could take full width and height of the visual area.

```suggestion
.River__visual--rounded,
.River__visual--rounded video,
.River__visual--rounded img {
  border-radius: var(--brand-borderRadius-large);
}
```